### ISSUE TITLE
Assess possible eddy currents in RF coils from TMS

### ISSUE THREAD

**Issue by BenInglis** [Brain-Imaging-Center/TMS-fMRI#9]
Some preliminary tests with the TMS coil inside the complete 32ch coil suggest that there may be some eddy currents being induced in conductive elements of the RF coil when the TMS coil is placed sufficiently close. In the preliminary tests the TMS coil was placed on top of a water phantom in the only remaining space, placing the TMS coil hard up against the upper inside surface of the RF coil. This is quite different to the typical location of the TMS coil inside the birdcage, where the larger volume means a greater distance from the TMS coil to the RF coil. So, the general questions is whether there are observable eddy current effects in any of our three RF coils. If there are, what are their magnitudes and durations? Can they be suppressed adequately by spacing of the TMS from the RF coil?

**Comment by DobyRahnev** [comment]
When the birdcage is back, perhaps Justin can run a test with the TMS coil up against the upper surface of the birdcage? In the meantime, perhaps an equivalent test with the 12-channel coil (rather than the birdcage) would also be useful?

Would the eddy currents depend on the timing of the firing of the TMS coil in respect to the EPI slice?

**Comment by BenInglis** [comment]
We have determined that there is strong coupling between the TMS pulses and any RF coil. The coupling will be addressed in a new issue. Similarly, we have also determined that the biphasic TMS pulse has a non-zero residual gradient area that is time-invariant, i.e. it isn't due to an eddy current but is actually an unbalanced positive and negative lobe on the biphasic TMS pulse. This issue will also be addressed in a new issue. 

This issue will be closed for now.
